samples/components/oauth/redshift.azure.xml

```diff
@@ -34,12 +37,17 @@
     </entry>
     <entry>
       <key>OAUTH_CAP_CLIENT_SECRET_IN_URL_QUERY_PARAM</key>
-      <value>true</value>
```

Comment:
for azure, i think we should set this as true? client id and client secret should be in url param: https://learn.microsoft.com/en-us/entra/identity-platform/v2-oauth2-auth-code-flow#request-an-authorization-code

Reply:
I tested it (just the flow would complete, I don't have Azure OAuth setup for Redshift). I don't see that it says it must be in a URL param, I see that it must be URL encoded. There are examples of passing in POST also.

Reply:
yeah i think either way should work, we have been using it in url param for a while: https://gitlab.tableausoftware.com/connectivity/plugins/azure-sqldb/-/blob/master/src/script/oauthConfig.xml?ref_type=heads#L45, but as long as you tested it we should be good i think